t/local/http.t

```diff
@@ -246,6 +246,31 @@ sub _test {
         isa_ok($res, 'HTTP::Response', 'basicAuth: good response object');
         is($res->code, 401, 'basicAuth: code 401');
     }
+    { # basic auth, UTF-8
+        my $req = HTTP::Request->new(GET => url("/basic_utf8", $base));
+        my $res = MyUA4->new->request($req);
+        isa_ok($res, 'HTTP::Response', 'basicAuthUtf8: good response object');
```

Comment:
is it possible to rename this (and others) to `'basicAuthUTF8'`, because `'basicAuthUtf8'` is hard to read

Reply:
Matter of taste, but I see your suggestion is more in line with what LWP otherwise uses, so changed.